src/benchmarks/micro/corefx/System.Memory/Constructors.cs

```diff
@@ -76,6 +76,12 @@ public Constructors()
         [Benchmark]
         public System.ReadOnlyMemory<T> ReadOnlyMemoryFromArrayStartLength() => new System.ReadOnlyMemory<T>(_nonEmptyArray, start: 0, length: Size);
 
+        [Benchmark]
+        public System.Span<T> ArrayAsSpan() => _nonEmptyArray.AsSpan();
```

Comment:
This does not cover the case that regressed in linked issue. The case that regressed was `AsSpan` overload that takes the index and length arguments.

Reply:
Also this is an array of 10 elements. @Anipik 's case was 100_000 elements. They may have different perf characteristics.